package.json

```
@@ -6,6 +6,7 @@
   "main": "src/index.js",
   "type": "module",
   "scripts": {
+    "dev": "nodemon ./src/dev.js localhost",
```

standart is:

```suggestion
    "start": "nodemon ./src/dev.js localhost",
```

so you can just use `npm start`

---

also, better put the nodemon config in it's own structure:

see https://github.com/adobe/helix-admin/blob/102a7c24bb57691f71c7cde9a26eb48457aa67e0/package.json#L134-L141

---

@tripodsan Got it, Thanks, will revert to leverage the main. I'll also integrate the franklin deploy dev server.

Can you give me access to `adobe/helix-admin` repo so I can compare and adjust with the preferred nodemon structure?